tests/functional/spec/thank_you.spec.js

```diff
@@ -53,10 +54,37 @@ describe("Thank You Default View Response Enabled", () => {
       $(DidYouKnowPage.submit()).click();
       $(ThankYouSubmitPage.submit()).click();
       expect(browser.getUrl()).to.contain(ThankYouPage.pageName);
+      expect($(ThankYouPage.viewSubmittedGuidance()).isDisplayed()).to.be.false;
       expect($(ThankYouPage.title()).getHTML()).to.contain("Thank you for completing the Test Thank You");
       expect($(ThankYouPage.viewAnswersTitle()).getHTML()).to.contain("Get a copy of your answers");
       expect($(ThankYouPage.viewAnswersLink()).getText()).to.contain("save or print your answers");
       expect($(ThankYouPage.viewSubmittedWarning()).getHTML()).to.contain("For security your answers will only be available to view for 45 minutes");
     });
+
+    it("When I navigate to the thank you page, and I have submitted more than 45 minutes ago, Then I shouldn't see the option to view my answers", () => {
+      $(DidYouKnowPage.yes()).click();
+      $(DidYouKnowPage.submit()).click();
+      $(ThankYouSubmitPage.submit()).click();
+      expect(browser.getUrl()).to.contain(ThankYouPage.pageName);
+      browser.pause(46000); // Waiting a minute for the timeout to expire
```

Comment:
46000 is less than a minute.

Reply:
I did initially have it at 60000 but changed it because the test was timing out. Ill update the comments